src/lib.rs

```diff
@@ -190,6 +197,27 @@ impl std::error::Error for ParseError {
     }
 }
 
+trait ToAttributeMaps {
+    fn to_attribute_maps(self) -> (AttributeMap<String, String>, AttributeMap<String, Namespace>);
+}
+
+impl ToAttributeMaps for Vec<xml::attribute::OwnedAttribute> {
+    fn to_attribute_maps(self) -> (AttributeMap<String, String>, AttributeMap<String, Namespace>) {
+        let mut attr_map = AttributeMap::with_capacity(self.len());
+        let mut attr_map_ns = AttributeMap::new();
+
+        for attr in self {
+            if let Some(ns) = attr.name.prefix {
+                let xns = Namespace::from_iter([(ns, attr.name.namespace.unwrap_or(xml::namespace::NS_EMPTY_URI.to_string()))]);
+                attr_map_ns.insert(attr.name.local_name.clone(), xns);
+            }
+            attr_map.insert(attr.name.local_name, attr.value);
+        }
+
+        (attr_map, attr_map_ns)
+    }
+}
+
```

Comment:
This is the important bit, most of the rest is tests/boilerplate to try to keep as much compatibility with the old non-newtype `xmltree::Namespace` as Rust will let us have.